test/mocha.opts

```diff
@@ -1,5 +1,6 @@
 --reporter spec
 --recursive
 --check-leaks
+--timeout 10000
```

Comment:
I don't think it has, no. The "external" tests started failing on CI with node.js 0.12 due to mocha timing out. Might just have been Travis that was temporarily overloaded, and this change made it go away while I was trying to fix the build for this branch.